### ISSUE TITLE
Ubuntu/Clang14 incompatibility.

### ISSUE THREAD

**Issue by evoskuil** [libbitcoin/libbitcoin-system#1380]
```
/usr/bin/../lib/gcc/x86_64-linux-gnu/13/../../../../include/c++/13/chrono:2320:48: error: call to consteval function 'std::chrono::hh_mm_ss::_S_fractional_width' is not a constant expression
        static constexpr unsigned fractional_width = {_S_fractional_width()};
                                                      ^
/usr/bin/../lib/gcc/x86_64-linux-gnu/13/../../../../include/c++/13/chrono:2320:48: note: undefined function '_S_fractional_width' cannot be used in a constant expression
/usr/bin/../lib/gcc/x86_64-linux-gnu/13/../../../../include/c++/13/chrono:2275:2: note: declared here
        _S_fractional_width()
        ^
  CXX      src/chain/libbitcoin_system_la-chain_state.lo
```
https://github.com/libbitcoin/libbitcoin-system/actions/runs/7681642280/job/20949297354?pr=1378#step:8:627

**Comment by evoskuil** [comment]
The problem is an incompatibility with Clang14.
https://github.com/actions/runner-images/issues/8659

**Comment by evoskuil** [comment]
https://github.com/libbitcoin/libbitcoin-system/pull/1378

**Comment by pmienk** [comment]
Addressed by [libbitcoin-build#305](https://github.com/libbitcoin/libbitcoin-build/pull/305) which bumps to clang-15.

**Comment by evoskuil** [comment]
Resolved
